### DISCUSSION TITLE
Welcome to Project-Pwnag0dchi Discussions!

### DISCUSSION THREAD

**Discussion by SHUR1K-N** [SHUR1K-N/Project-Pwnag0dchi#1] [General]
## Welcome!
Here, you can:
  * Ask questions you’re wondering about
  * Share ideas
  * Engage with other community members

  To get started, comment below!

**Comment by dleerdefi** [comment]
This is an epic build! Absolutely badass! 

I followed the steps and when I rebooted my pwnagotchi, the new interface loaded but instantly freezes. I see "100%" under cpu so I am wondering if the startup is too taxing on my device? 

I was able to login to the web interface and see my new plugins but when I try to reboot to Auto the freezing issue ensues. Any idea on best ways to troubleshoot? 

Thanks!

**Reply by SHUR1K-N** [reply]
Thanks! Opened an Issue (#2) in reference to your comment; let's troubleshoot there. 🙂

**Comment by kornhn** [comment]
Hi, I just finished the pwnagotchi with all your plugins but I noticed that it gets too hot.
hola acabo de terminar el pwnagotchi con todo tus plugins pero e notado que se caleinta demasiado

**Reply by SHUR1K-N** [reply]
How hot? What does memtemp-plus say? All people I tested this project with have reported 40-50°C (which is normal).

**Comment by Draco1704** [comment]
Hi, after updating my recent gotchi with all these cool plugins I noticed that the handshakes-dl plugin does open the list of APs but when I try to download a sigle one it gives me an internal server error (500), but I can still download all as a zip, also the exp plug in does't show when I turn it on, I'm also not sure if the display-password plugin is working as it does not show anything.
Do I need to delete the previous handshakes my gotchi captured previous to the plugin instalations??

**Comment by Narfianer** [comment]
Hello,

i am currently trying to configure my Pwnagotchi but am encountering some problems and hope for your support.

Problems:

Missing Plugins in the Web Interface:

Although I can see all plugins in the backend and have the correct plugins installed, they are not displayed in the web interface, so I can't activate them. Specifically, the better_quickdic plugin is missing. Overall, the plugins visible are different from those in the YouTube tutorial I followed.

Auto Mode Not Working When Connected via USB:

When I connect the Pwnagotchi via the connect USB port and am connected through the web interface, I can't switch it to auto mode.
It restarts, goes into auto mode on the PaperInk display, but after a few seconds, it restarts again and switches to menu mode.

What I've Done So Far:

Flashed the image as described in the YouTube tutorial.
Successfully set up web access.

Question:

Has anyone experienced similar issues or have any ideas on how to fix these problems?

Thank you in advance for your help!

**Reply by SHUR1K-N** [reply]
Are you using your own `config.toml`, or the one from this project?

**Reply by Narfianer** [reply]
that from the project. I changed my SSID, generated and entered the key and changed the password for the web login. Otherwise everything is the same.

**Reply by SHUR1K-N** [reply]
Could you open an Issue and put your `config.toml` in it? Don't forget to remove your wpa-sec API key and web UI credentials.

**Reply by Narfianer** [reply]
https://github.com/SHUR1K-N/Project-Pwnag0dchi/issues/12

**Reply by Narfianer** [reply]
I'm new to git, did I open the issue correctly?

**Comment by ApexXP** [comment]
Hi, When I try  to transfer the addons with fliezilla it says that it could not start the transfer. Do you think you could help with this.

**Reply by ApexXP** [reply]
I  just wached you video on the custom faces and that fixed it

**Comment by gsp2000** [comment]
First and foremost, THANK YOU!!! Awesome repo, great tutorials! I have a question hardware related. Built two of these based on the latest Jay image and your tutorial. One is on a rpi zero 2w, the other on the older (original) rpi0 w. The latter seems to be giving me some issues. Are there any known tweaks for the rpi0? Reimaged it twice and its just not working right. Slow response time, offline dictionary doesnt seem to work and so on. Any chance I can get this to work right or should I just get another rpi0 2? 

Edit: The plugins giving me issues are better_quickdic (no matter what I tried, I wont get the cracked file) and the handshake_dl (gets a server busy error when dl an individual handshake)
